projects/storefrontapp-e2e-cypress/cypress/helpers/anonymous-consents.ts

```diff
@@ -1,24 +1,42 @@
 import { giveConsent } from '../helpers/consent-management';
 import { standardUser } from '../sample-data/shared-users';
+import { switchSiteContext } from '../support/utils/switch-site-context';
 import { login } from './auth-forms';
 import { registerUser, waitForPage } from './checkout-flow';
 import { checkBanner } from './homepage';
 import { signOutUser } from './login';
-import {
-  LANGUAGES,
-  LANGUAGE_DE,
-  LANGUAGE_LABEL,
-  LANGUAGE_REQUEST,
-  siteContextChange,
-  stub,
-} from './site-context-selector';
+import { LANGUAGE_DE, LANGUAGE_LABEL } from './site-context-selector';
 import { generateMail, randomString } from './user';
 
 const ANONYMOUS_BANNER = 'cx-anonymous-consent-management-banner';
 const ANONYMOUS_DIALOG = 'cx-anonymous-consents-dialog';
+const GIVEN = 'ON';
+const WITHDRAW = 'OFF';
+const ALLOW_ALL = 'Allow All';
+const REJECT_ALL = 'Reject All';
+const BE_CHECKED = 'be.checked';
+const NOT_BE_CHECKED = 'not.be.checked';
+
+//TODO: improve anonymous consents e2e test
```

Comment:
No need for the `TODO:`, you can remove it.
Maybe the message can say: As the consent state is synchronized with the back-end in a stateless way (using an http header) we have to wait for the last one _XHR_ request to finish before mutating the ngrx state; otherwise, the state would be overridden by `anonymous-consents-interceptor.ts`. Currently, there's no support for this, thus the `cy.wait(5000)`